content/en/tracing/troubleshooting/_index.md

```diff
@@ -164,13 +164,22 @@ var tracer = Tracer.Create(isDebugEnabled: true);
 Tracer.Instance = tracer;
 ```
 
+The environment variable `DD_TRACE_DEBUG` can also be set to `true`.
```

Comment:
Oh, good catch, @wantsui. The code above (`Tracer.Create(isDebugEnabled: true)`) has been deprecated for some time. It still works, but only for backwards compatibility, so we don't break existing users.

Going forwards, the best way to enable debug mode in the Tracer is setting `DD_TRACE_DEBUG` via one of the available [configuration options](https://docs.datadoghq.com/tracing/setup/dotnet#configuration) (env vars, `app.config`/`web.config`, json files, or in code via `TraceSettings.DebugEnabled`). This applies to both manual and automatic instrumentation.

To enable debug mode in the Profiler (`dotnet-profiler.log`), `DD_TRACE_DEBUG` only works as an environment variable. This applies to auto-instrumentation only

Reply:
Should `Tracer.Create(isDebugEnabled: true)` be kept in the docs? Or should it be removed in favor of explaining how to configure the  `DD_TRACE_DEBUG` setting?

Reply:
I should be removed in favor of the newer [configuration system](https://docs.datadoghq.com/tracing/setup/dotnet#configuration).

Reply:
Is it okay if that gets added in a future update? I still have a few questions of what that'd look like.